app/assets/stylesheets/content/_timelines.scss

```diff
@@ -0,0 +1,137 @@
+//-- copyright
+// OpenProject is a project management system.
+// Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License version 3.
+//
+// OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+// Copyright (C) 2006-2013 Jean-Philippe Lang
+// Copyright (C) 2010-2013 the ChiliProject Team
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+//
+// See doc/COPYRIGHT.rdoc for more details.
+//++
+
+#content table.issues td.center,
+#content table th.center {
+  text-align: center;
+}
+.contextual .new-element {
+  background: url("../images/add.png") no-repeat scroll 6px center transparent;
+  padding-left: 16px;
+  font-size: 11px;
+}
+
+.timelines-contextual-fieldset {
+  float:right;
+  white-space:nowrap;
+  margin-right:24px;
+  padding-left:10px;
+}
+
+.timelines-milestone, .timelines-phase {
+  border: 1px solid #000;
+  display: inline-block;
+  height: 12px;
+  width: 12px;
+  margin-bottom: -3px;
+  margin-right: 5px;
+}
+.timelines-milestone {
+  transform: rotate(45deg);
+}
+.timelines-phase {
+  border-radius: 4px;
+}
+.timelines-attributes th {
+  width: 17%;
+}
+.timelines-attributes td {
+  width: 33%;
+}
+#content .meta table.timelines-attributes th,
+#content .meta table.timelines-attributes td {
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+#content table.timelines-rep,
+table.timelines-dates {
+  width: 100%;
+}
+#content table.timelines-dates th {
+  text-align: left;
+  padding-left: 1em;
+  padding-right: 1em;
+}
+table.timelines-dates td {
+  padding: 1em;
+}
+
+table.timelines-dates tbody.timelines-current-dates td {
+  font-weight: bold;
+  height: 3em;
+  vertical-align: middle;
+  padding: 0 1em;
+}
+
+#content table.timelines-pt td, #content table.timelines-pt th,
+#content table.timelines-pet td, #content table.timelines-pet th {
+  border: 1px solid #E6E6E6;
```

Comment:
Color literals like `#E6E6E6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E6E6E6` should be written as `#e6e6e6`<br>Each selector in a comma sequence should be on its own line